docs/pom.xml

```diff
@@ -98,9 +52,11 @@
 						<artifactId>git-commit-id-plugin</artifactId>
 					</plugin>
 					<plugin>
+						<groupId>org.apache.maven.plugins</groupId>
```

Comment:
Just confirming, I saw that the `maven-dependency-plugin` and `maven-resources-plugin` were missing the `groupId`s; Are these plugins necessary for the docs gen to work?

Reply:
Not sure - it was part of a copy/paste from https://github.com/spring-cloud/spring-cloud-build#reusing-the-documentation. I didn't test it without the group IDs